### PR TITLE
fix some text

### PR DESCRIPTION
**Pull request by vincentmoreaudba**
# PR Description

Fix some text 

## Link to the initial request/ticket

- Fixes #.

## What this PR Provides

- describe in short sentences the goal of the PR. use lists.

## Known limitations / corner cases (if any)

- if none, use `N/A`

## How to test

- give what we need to test. example give the sql script to create the context.
  
### Expected results

- describe the expected result after a run

### CODE REVIEW

**Review by LeopoldPetit — APPROVED** [review]
legit

**Review by LeopoldPetit — APPROVED** [review]
legit

**Review by arkram — APPROVED** [review]

**Review by regislegrand — APPROVED** [review]

File: tests/rules/S001/test_SchemaWithDefaultRoleNotGranted.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def test_schema_with_default_role_granted_on(postgres_instance_args) -> None:
     )
     sarif_document = SarifDocument()
     context = Context(
-        desc="The schema ha no default role. Means that futur table will not be granted through a role. So you will have to re-execute grants on it.",
+        desc="The schema has no default role. It means that future tables access will not be granted through a role. So you will have to re-execute grants on it.",
         fixes=[
             "add a default privilege=> ALTER DEFAULT PRIVILEGES IN SCHEMA <schema> for user <schema's owner>",
```
Comment (foezor) [inline tests/rules/S001/test_SchemaWithDefaultRoleNotGranted.py]:
```suggestion
            "add a default privilege => ALTER DEFAULT PRIVILEGES IN SCHEMA <schema> for user <schema's owner>",
```

File: tests/data/good_config.yaml [hunk 2]
```diff
@@ -252,7 +252,7 @@ schema:
     ruleid: S001
     enabled: True
     context:
-      desc: The schema ha no default role. Means that futur table will not be granted through a role. So you will have to re-execute grants on it.
+      desc: The schema has no default role. It means that future tables access will not be granted through a role. So you will have to re-execute grants on it.
       message: "No default role granted on schema {0}.{1}. It means that each time a table is created, you must grant it to roles."
       fixes:
         - "Add a default privilege=> ALTER DEFAULT PRIVILEGES IN SCHEMA <schema> for user <schema's owner>"
```
Comment (foezor) [inline tests/data/good_config.yaml]:
```suggestion
        - "Add a default privilege => ALTER DEFAULT PRIVILEGES IN SCHEMA <schema> for user <schema's owner>"
```

File: tests/data/config_file_wrong_function_name.yaml [hunk 3]
```diff
@@ -108,6 +108,6 @@ schema:
   - name: SchemaWithDefaultRoleNotGranted
     enabled: True
     context:
-      desc: The schema ha no default role. Means that futur table will not be granted through a role. So you will have to re-execute grants on it.
+      desc: The schema has no default role. It means that future tables access will not be granted through a role. So you will have to re-execute grants on it.
       fixes:
         - add a default privilege=> ALTER DEFAULT PRIVILEGES IN SCHEMA <schema> for user <schema's owner> GRANT SELECT ON TABLES TO <schema>_read_only
```
Comment (foezor) [inline tests/data/config_file_wrong_function_name.yaml]:
```suggestion
        - add a default privilege => ALTER DEFAULT PRIVILEGES IN SCHEMA <schema> for user <schema's owner> GRANT SELECT ON TABLES TO <schema>_read_only
```

**Review by foezor — APPROVED** [review]
minor comments not mandatrory to treat, just in case

**Review by everleene — APPROVED** [review]
LGTM
